apis/Google.LongRunning/Google.LongRunning/OperationsClientPartial.cs

```diff
@@ -34,6 +35,19 @@ public virtual IScheduler Scheduler
         {
             get { throw new NotImplementedException(); }
         }
+
+        // It would be nice to make this internal, but then anyone creating their
+        // own subclass for testing would be broken.
```

Comment:
I'm not sure I understand exactly why that would break libraries with their own subclasses, but would protected internal work as well?